Dockerfile.test.cpu

```diff
@@ -10,7 +10,7 @@ ARG TENSORFLOW_PACKAGE=tensorflow==1.14.0
 ARG KERAS_PACKAGE=keras==2.2.4
 ARG PYTORCH_PACKAGE=torch==1.2.0+cpu
 ARG TORCHVISION_PACKAGE=torchvision==0.4.0+cpu
-ARG MXNET_PACKAGE=mxnet==1.5.0
+ARG MXNET_PACKAGE=mxnet==1.6.0
```

Comment:
Hey @eric-haibin-lin, this version is overridden by the settings in https://github.com/horovod/horovod/blob/master/docker-compose.test.yml#L90.

To update, you can do the following:

1. Change `MXNET_PACKAGE` version for each test you want to update (I recommend keeping some of the tests as 1.5 to improve test coverage, preferably the ones that also have older TF/PyTorch versions).
2. Update the test name to `mxnet_1_6_0` as shown here: https://github.com/horovod/horovod/blob/master/docker-compose.test.yml#L79
3. Update the test name in `gen-pipeline.sh`: https://github.com/horovod/horovod/blob/master/.buildkite/gen-pipeline.sh#L16
4. Run `BUILDKITE_PIPELINE_SLUG=SLUG BUILDKITE_BRANCH=BRANCH .buildkite/gen-pipeline.sh > test/data/expected_buildkite_pipeline.yaml` to update expected buildkite generated script.